17:37:49 - [36mINFO [0m hunit_test.py setUp:912            
################################################################################
Test_pytest_repro1.test_classes1
################################################################################
17:37:49 - [36mINFO [0m lib_tasks_pytest.py pytest_repro:830                   Reading file_name='/app/helpers/test/outcomes/Test_pytest_repro1.test_classes1/tmp.scratch/cache/lastfailed'
17:37:49 - [36mINFO [0m lib_tasks_pytest.py pytest_repro:832                   Reading failed tests from file '/app/helpers/test/outcomes/Test_pytest_repro1.test_classes1/tmp.scratch/cache/lastfailed'
17:37:49 - [33mWARN [0m lib_tasks_pytest.py pytest_repro:876                   Skipping test='dev_scripts/testing/test/test_run_tests.py' since test_file_name='dev_scripts/testing/test/test_run_tests.py', test_class=''
17:37:49 - [33mWARN [0m lib_tasks_pytest.py pytest_repro:876                   Skipping test='dev_scripts/testing/test/test_run_tests2.py' since test_file_name='dev_scripts/testing/test/test_run_tests2.py', test_class=''
17:37:49 - [33mWARN [0m lib_tasks_pytest.py pytest_repro:876                   Skipping test='documentation/scripts/test/test_all.py' since test_file_name='documentation/scripts/test/test_all.py', test_class=''
17:37:49 - [33mWARN [0m lib_tasks_pytest.py pytest_repro:876                   Skipping test='documentation/scripts/test/test_render_md.py' since test_file_name='documentation/scripts/test/test_render_md.py', test_class=''
17:37:49 - [36mINFO [0m lib_tasks_pytest.py pytest_repro:943                   Found 3 failed pytest 'classes' target(s); to reproduce run:
pytest helpers/test/test_printing.py::Test_dedent1
pytest helpers/test/helpers/test/test_list.py::Test_list_1
pytest helpers/test/test_cache.py::TestAmpTask1407
[95m## pytest_repro: [0m
# To run the tests:
> ./tmp.pytest_repro.sh
